infrastructure/gsuite-exporter/gsuite_exporter/cli.py

```diff
@@ -0,0 +1,108 @@
+
+def get_exporter_cls(exporter_class):
+    try:
+        file, name = tuple(exporter_class.split('.'))
+        package = 'gsuite_exporter.exporters.%s' % file
+        return getattr(importlib.import_module(package), name)
+    except Exception as e:
+        logger.exception(
+            'Exporter "%s" not found. Verify the exporter class name is'
+            'prefixed by the exporter file name.\n'
+            'Example: `stackdriver_exporter.StackdriverExporter`',
+            exporter_class)
+        sys.exit()
+
+def sync_all(
+        admin_user,
+        api,
+        applications,
+        project_id,
+        exporter_cls,
+        credentials_path=None):
+    """Query last data from Admin SDK API and export them to the destination.
+
+    Args:
+        credentials_path (str): The GSuite Admin credentials file.
+        api (str): The GSuite Admin API to get data from.
+        token_path (str): The GSuite Admin Token file.
+        applications (list): The Gsuite Admin Applications to query.
+        project_id (str): The project id to export the data to.
+        exporter_cls (str): The exporter class to use.
+    """
+    fetcher = AdminReportsAPIFetcher(admin_user, credentials_path)
+    exporter = get_exporter_cls(exporter_cls)(
+        project_id=project_id,
+        credentials_path=credentials_path)
+
+    for app in applications:
+        last_timestamp = exporter.get_last_timestamp(app)
+        exporter_dest = exporter.get_destination(app)
+        logger.info(
+            "%s.%s --> %s (%s) [starting new sync]. Last sync: %s",
+            api,
+            app,
+            exporter_cls,
+            exporter_dest,
+            last_timestamp)
+
+        records_stream = fetcher.fetch(
+            application=app,
+            start_time=last_timestamp)
+
+        for records in records_stream:
+            response = exporter.send(records, app, dry=False)
+            logger.info(
+                "%s.%s --> %s (%s) [%s new records synced]",
+                api,
+                app,
+                exporter_cls,
+                exporter_dest,
+                len(records))
+
+def main():
+    parser = argparse.ArgumentParser(description='Add some integers.')
+    parser.add_argument('--admin-user', type=str, help='The GSuite Admin user email.', required=True)
+    parser.add_argument('--api', type=str, help='The GSuite Admin API.', default='reports_v1', required=False)
+    parser.add_argument('--applications', type=str, nargs='+', help='The GSuite Admin Applications.', required=True)
+    parser.add_argument('--project-id', type=str, help='The project id to export GSuite data to.', required=True)
+    parser.add_argument('--exporter', type=str, help='The exporter class to use.', default='stackdriver_exporter.StackdriverExporter', required=False)
+    parser.add_argument('--credentials-path', type=str, help='The service account credentials file.', default=None, required=False)
```

Comment:
Line length > 80

Reply:
I personally always maintain line length, and the Google style docs are pretty clear that it should be limited to 80 chars.

I typically put `help` last so I can wrap it as necessary to meet line length.  Style docs prefer parens for implicit string joining to meet line length requirements.

I typically do this for each
```
parser.add_argument(
    '--credentials-path', 
    type=str, 
    default=None, 
    required=False,
    help='The service account credentials file.', 
)
````

Reply:
Fixed.